src/sidebar/components/PendingUpdatesNotification.tsx

```diff
@@ -89,25 +119,26 @@ function PendingUpdatesNotification({
         onClick={applyPendingUpdates}
         unstyled
         classes={classnames(
-          'flex gap-1.5 items-center py-1 px-2',
+          'flex items-center',
           'rounded shadow-lg bg-gray-900 text-white',
           'group focus-visible-ring',
         )}
+        data-testid="notification"
+        data-collapsed={collapsed}
       >
-        <DownloadIcon className="w-em h-em opacity-80" />
-        <div
-          data-testid={
-            collapsed ? 'collapsed-notification' : 'full-notification'
-          }
-          className="flex gap-1"
-        >
-          <HideableBlock hidden={collapsed}>Load</HideableBlock>
-          <span className="font-bold">{pendingUpdateCount}</span>
-          <HideableBlock hidden={collapsed}>
-            {pluralize(pendingUpdateCount, 'update', 'updates')}
-          </HideableBlock>
-        </div>
-        <span className="sr-only">by pressing l</span>
+        <UpdateBlock
+          type="updates"
+          collapsed={collapsed}
+          count={pendingUpdateCount}
+        />
+        {pendingMentionCount > 0 && (
+          <UpdateBlock
+            type="mentions"
+            collapsed={collapsed}
+            count={pendingMentionCount}
+          />
+        )}
```

Comment:
In a follow-up PR we may change this so that every block is an individual button.